.prettierrc.mjs

```diff
@@ -1,8 +1,11 @@
 // All of these are defaults except singleQuote, but we specify them
 // for explicitness
-module.exports = {
+const config = {
```

Comment:
nit: Do you know if the above comment still applies?

> // All of these are defaults except singleQuote, but we specify them
// for explicitness

Reply:
The values specified are still the defaults, with the exception of the `plugins` item (required for Prettier 3). We don't specify all options, but I'm not sure if that's necessary.